spec/README.md

```diff
@@ -11,4 +11,5 @@ Here is an overview:
     - [Azure Service Bus Queue/Topic](./triggers/azure-service-bus.md)
     - [Azure Storage Queue](./triggers/azure-storage-queue.md)
     - [RabbitMQ Queue](./triggers/rabbit-mq-queue.md)
-    - [Redis](./triggers/redis.md)
\ No newline at end of file
+    - [Redis](./triggers/redis.md)
+    - [GCP Pub/Sub](./triggers/gcp-pub-sub.md)
```

Comment:
Nitpicking but can you add it to the list in alphabetical order please?

Reply:
@tomkerkhove this is fixed now.

Reply:
Thanks!